openerp/addons/base/migrations/9.0.1.3/pre-migration.py

```diff
@@ -54,7 +54,23 @@
 }
 
 
+OBSOLETE_RULES = (
+    'multi_company_default_rule',
+    'res_currency_rule',
+)
+
+
+def remove_obsolete(cr):
+    openupgrade.logged_query(cr, """
+        delete from ir_rule rr
+        using ir_model_data d where rr.id=d.res_id
+        and d.model = 'ir.rule' and d.module = 'base'
+        and d.name in {}
+        """.format(OBSOLETE_RULES))
```

Comment:
@coleste why do you need to do this manually? Shouldn't Odoo clean that up at the end of the migration run?